src/linux/cgroups2.cpp

```diff
@@ -795,6 +795,7 @@ Result<Bytes> parse_bytelimit(const string& value)
 namespace control {
 
 const string CURRENT = "memory.current";
+const string HIGH = "memory.high";
 const string MAX = "memory.max";
 const string MIN = "memory.min";
```

Comment:
we can add all the control functionality here, but it looks like we're only going to use memory.low and memory.max: https://facebookmicrosites.github.io/cgroup2/docs/memory-controller.html

How crun handles requests vs limits:
https://github.com/containers/crun/blob/68df0832427e692c705bd3485719174091b37cb5/crun.1.md#memory-controller

This is how mesos would do request vs limits as well for burstable containers:
https://mesos.apache.org/documentation/latest/running-workloads/#resource-requests-and-limits